### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by programad** [Azure/pipelines#17]
Hey, everyone. thank you for this github action.

Is there a way to pass a parameter to the Azure Pipeline we want to call??

I want to call an Azure Pipeline only after the semantic release is generated here on github so when calling the Azure Pipeline, I want to tell it to use an specific variable value containing the latest github release generated to put a footer information on my website.(eg.: v1.5.12)

Again, thank you for this.

**Comment by LTA-Thinking** [comment]
+1. I have a similar request. The pipeline I'm triggering has a variable I want to set when triggering it.

**Comment by richardsimko** [comment]
+1 also looking to be able to set a variable in the pipeline build since we want to use this for triggering releases from the GH Deployments.

**Comment by danquah** [comment]
👋  @N-Usha - any word on when eg. https://github.com/Azure/pipelines/pull/18 or https://github.com/Azure/pipelines/pull/11 could get merged? We've had to maintain a fork of this action for several months now, and speaking frankly I went from being very happy to find this action which demonstrates willingness to integration Github and DevOps Pipelines, to now being quite disappointed to see two seperate contributions of a rather small and valuable feature go stale :(

**Comment by samcarswell** [comment]
+1 We would also find this useful.

**Comment by mikkolaakko** [comment]
+1

Looking a way to run Azure DevOps pipeline when there is a new pull request in GitHub, or a change in any of the feature branches.

**Comment by ds-ms** [comment]
@N-Usha could you take a look?
